pkg/test/env/istio.go

```diff
@@ -72,7 +72,7 @@ var (
 	// IstioTop has the top of the istio tree, matches the env variable from make.
 	IstioTop = TOP.ValueOrDefaultFunc(getDefaultIstioTop)
 
-	// IstioSrc is the location if istio source ($TOP/src/istio.io/istio
+	// IstioSrc is the location of istio source ($TOP/src/istio.io/istio
 	IstioSrc = path.Join(IstioTop, "src/istio.io/istio")
```

Comment:
this doesn't work properly locally. istioSrc locally should be /work in all cases. Its just non-locally that IstioSrc may work in this fashion. Looking at the comment in linen 75, this makes sense.